src/main/java/edu/ucsb/cs156/dining/controllers/ReviewsController.java

```diff
@@ -0,0 +1,100 @@
+package edu.ucsb.cs156.dining.controllers;
+
+import edu.ucsb.cs156.dining.entities.User;
+import edu.ucsb.cs156.dining.entities.Reviews;
+import edu.ucsb.cs156.dining.models.CurrentUser;
+import edu.ucsb.cs156.dining.entities.UCSBDiningCommons;
+import edu.ucsb.cs156.dining.errors.EntityNotFoundException;
+import edu.ucsb.cs156.dining.repositories.ReviewsRepository;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import lombok.extern.slf4j.Slf4j;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.security.core.userdetails.UserDetails;
+
+import org.springframework.security.web.bind.annotation.AuthenticationPrincipal;
+
+import java.util.ArrayList;
+import jakarta.validation.Valid;
+
+/**
+ * This is a REST controller for Reviews
+ */
+
+@Tag(name = "Reviews")
+@RequestMapping("/api/reviews")
+@RestController
+@Slf4j
+public class ReviewsController extends ApiController {
+
+    @Autowired
+    ReviewsRepository reviewsRepository;
+
+    /**
+     * THis method returns a list of all reviews.
+     * @return a list of all reviews
+     */
+    @Operation(summary= "List all ucsb reviews of dining commons")
+    @PreAuthorize("hasRole('ROLE_USER')")
```

Comment:
```suggestion
    @PreAuthorize("hasRole('ROLE_ADMIN')")
```

See more comments below.